best-practices/rspec.md

```diff
@@ -0,0 +1,30 @@
+## Rspec style guide
+
+- First #describe what you are doing
+- Then establish the #context
+- #it only expects one thing (within reason)
+- Prefer explicitness - #it #its #specify #subject are nice shortcuts but
+consider whether they are sacrificing readability
```

Comment:
At the same time, `it "works"` is not as concise as `specify`. If you've declared your expectation in english in the `describe` above (if you've done so in the `context`, you're doing `context` wrong), use specify, or move it into the `it` argument.